Merged forgotten relics into Advanced botany

EnderProyects opened the pull request:
Add thaum/botania compatibility in recipes, and post gaia recipes, will need help of @combusterf or @Alastors to nerf/buff recipes, it adds utility items, that uses endgame matelias of both thaum and botania
![image](https://user-images.githubusercontent.com/45178400/226943976-82e2a019-7ecd-4451-b6ad-530061bbe227.png)

EnderProyects committed:
Try to merge FR

Try to merge Forgotten relics

EnderProyects committed:
Fix issues

EnderProyects committed:
Fix2

combusterf commented:
Copy-pasting one entire repo into another is not an appropriate way of sneaking in an entire new mod into the modpack - its better to fork those repos separately.

Also, please get consensus and approval here first https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11517

boubou19 commented:
converted to draft until consensus and approval are met

EnderProyects commented:
> Copy-pasting one entire repo into another is not an appropriate way of sneaking in an entire new mod into the modpack - its better to fork those repos separately.
> 
> Also, please get consensus and approval here first [GTNewHorizons/GT-New-Horizons-Modpack#11517](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11517)

There is a discord conversation of why this can be done, since the addon is DWTFYWPLicense I try to expand some things of the addon, and in any moment I said I was sneaking the mod in, the pr is named megerd one mod into another

EnderProyects committed:
Integration

Add a recipe for direct terrasteel block, add names of recipes for easy search, integrate recipes into gtnh using tools, nerf some endgame recipes

EnderProyects committed:
Merge branch 'master' of https://github.com/EnderProyects/Advanced-Botany
